verticapy/tests/utilities/test_utilities.py

```diff
@@ -56,6 +56,11 @@ def world_vd(base):
 
 
 class TestUtilities:
+    def test_print_display(self, titanic_vd):
+        print(titanic_vd)
+        display(titanic_vd)
+        assert titanic_vd.shape() == (1234, 14)
```

Comment:
It doesn't seem that this function tests `print` or `display`. The `assert` line is irrelevant to the two lines above it.

Reply:
Corrected - test deleted and one for the vdf repr was created